extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -341,6 +344,10 @@ export class AmpIframe extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    if (this.loadPromise_) {
```

Comment:
hmm, I think a fix for this needs to be in runtime. We don't wanna change every component to do this. Runtime does have enough information to decide whether to call `layoutCallback` or not. @jridgewell